apps/src/panels/panels.module.scss

```diff
@@ -0,0 +1,93 @@
+@import "color.scss";
+
+// We will fade in the next button for 1s after a 6s delay, then leave it showing.
+@keyframes fade-in {
+  0%, 50% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.panelsContainer {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  flex-direction: column;
+
+  .panel {
+    margin: 0 auto;
+    width: calc(100% - 40px);
+    height: calc(100% - 40px);
```

Comment:
Not sure if some of these values (ex. 40px) are meant to be the same in PanelsView, but if so, you could `:export` them from here and reference in the component to have a single source of truth

Reply:
Oh good point.  These lines were actually no longer needed (since code calculates these values) so I've removed.  I don't think there are other constants to share at this point.